conda/core/subdir_data.py

```diff
@@ -553,6 +554,11 @@ def _get_base_url(self, repodata: dict, with_credentials: bool = True) -> str:
         return self.url_w_subdir
 
 
+@deprecated(
+    "24.9",
+    "25.3",
+    addendum="Use `conda.core.models.records.PackageRecord.make_feature_record` instead.",
```

Comment:
I am wondering whether we should really provide this new API at all given how "in-practice-deprecated" features have been for a while.

Reply:
We (i.e. Jaime) noticed that the classic solver still needs it, so for now, it stays.